### ISSUE TITLE
Request: Mind over Matter calculations

### ISSUE THREAD

**Issue by Dagda47** [Openarl/PathOfBuilding#578]
I would like to see how optimal my MoM setup is. If you could add the optimal max life/mana under calculations one could easily see that there are improvements to make (for a given mana pool the life difference from the optimum and vice versa).

**Comment by qikink** [comment]
If you go to the "Calcs" section, on the right you can see "Life Protected" when you mouse over Mind Over Matter in the damage taken section
![mom](https://user-images.githubusercontent.com/2025166/29528801-258b8724-8652-11e7-9312-6cf6c378a46d.png)
. 

I'm not certain this is what you meant, but it should help.

**Comment by Dagda47** [comment]
Yeah i know, but you can't see the requested optima. With the informations currently provided i don't know if i have enough mana or life to maximize a hit that i could withstand.

**Comment by kazuoteramoto** [comment]
If the value in "Total life protected" is equal to your unreserved life you have the optimal setup. If is less you need more mana. Is this the information you want?

**Comment by Dagda47** [comment]
oh, almost. An adittional line saying how much mana i'm missing to protect my unreserved life or how much mana exceeds the optimum. So i know without further calculations/trial and error how much mana i need.
